tests/python-pytest/onnx/export/backend.py

```diff
@@ -0,0 +1,98 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
it seems to be added already, but what is the name `python-pytest`? 
there's already a folder `tests/python/unittest`

Reply:
mxnet unittests uses nosetests but onnx backend test framework uses pytest, so keep them separate we created another folder for pytests.

Reply:
just remove the python-pytest folder, and use onnx, the name is pretty confusing and meaningless as an empty one containing only onnx.

Reply:
in the future if there is another component that is built into MXNet that uses pytest instead of nosetests, then what will we do?

the point of naming it python-pytest is separate them from the other tests which uses nosetests framework? 

And this naming was part of a previous PR https://github.com/apache/incubator-mxnet/pull/9963 and was suggested by @marcoabreu during the review process.